tests/unittests/Foundation/NSURLTests.m

```diff
@@ -206,7 +206,8 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
     ASSERT_OBJCEQ_MSG(fileURLString, newFileURLString, "File URLs do not match!");
 }
 
-TEST(NSURL, CheckResourceIsReachable) {
+// Reference platform has bug on some machines where currentDirectoryPath returns the incorrect path
```

Comment:
it literally is _the path to the current directory_; if this test is based on the folder from which you're running it, the test is incorrect.